src/App.js

```diff
@@ -1,5 +1,8 @@
 import React from "react";
-import { SafeAreaInsetsContext, SafeAreaProvider } from 'react-native-safe-area-context';
+import {
+  SafeAreaInsetsContext,
+  SafeAreaProvider,
+} from "react-native-safe-area-context";
 
 import Root from "./containers";
 import "./App.css";
```

Comment:
This doesn't make any sense. **only spaces have been added** all over the place.